alua.c

```diff
@@ -319,6 +323,8 @@ tcmu_get_tgt_port_grp(struct tcmu_device *dev, const char *name)
 	free(orig_str_val);
 	return group;
```

Comment:
These patches are nice for future kernel changes but this is not possible right now is it?

The kernel always returns one of the checked values? If not then, we are missing some checks for valid values.